app/views/applications/_show.html.haml

```diff
@@ -11,7 +11,11 @@
       Email
       %small (required)
     %td= @user.email
-
+  %tr
+    %td.left
+      Preferred Pronouns
```

Comment:
Could delete "Preferred" for simplicity and directness without loss of generality